### DISCUSSION TITLE
Email receipt

### DISCUSSION THREAD

**Discussion by trymeouteh** [MrNaif2018/bitcart#283] [Ideas]
Please add an optional field for the customer to enter their email address for physical transactions to receive a digital receipt of the purchase.

Same for online transactions but for the option of the online store to automatically use the email used in the shipping info, billing info or in the users account.

And a copy of the receipt will be sent to the store to have a copy in case the customer returns the item.

**Comment by MrNaif2018** [comment]
Hi! Not sure if it is fully what you want, but the customer can already enter their email: in checkout page from admin panel and store. If store has configured email server, notifications will be sent to customer with the contents defined by email templates (defaults are a summary of items bought, most basic)

**Reply by trymeouteh** [reply]
For physical stores, when is the email field prompted? After payment or before payment will bitcartcc ask for the customer email?

If it emails a list of items and the prices with a unique ID and the date, that would work as a receipt even if it not meant to be a receipt.

**Reply by MrNaif2018** [reply]
Email is asked before payment in admin panel's checkout page if it is enabled in checkout settings of a store, in store POS there is email field too. 

By default the email sent doesn't contain invoice id or anything like so, but via custom templates you can create a message that includes {{invoice.id}} for example

**Reply by trymeouteh** [reply]
I see, my feature request/idea would be a setting for each store to enable/disable emailed receipts after a purchase is made. When this setting is enabled, once a customer enters their email and conducts the transactions, it will send a receipt email to the customers email and save this invoice onto the admin panel of the store.

A simple setting to enable instead of having to create a template.

**Reply by MrNaif2018** [reply]
Right now it works this way: if email server is configured and working-send email, otherwise no
